UC4-chaincode/src/main/java/de/upb/cs/uc4/chaincode/model/admission/AbstractAdmission.java

```diff
@@ -53,49 +47,18 @@ public void setEnrollmentId(String enrollmentId) {
         resetAdmissionId();
     }
 
-    /**
-     * Get courseId
-     *
-     * @return courseId
-     **/
-    @ApiModelProperty()
-    public String getCourseId() {
-        return this.courseId;
-    }
-
-    public void setCourseId(String courseId) {
-        this.courseId = courseId;
-        resetAdmissionId();
-    }
-
-    /**
-     * Get moduleId
-     *
-     * @return moduleId
-     **/
-    @ApiModelProperty()
-    public String getModuleId() {
-        return this.moduleId;
-    }
-
-    public void setModuleId(String moduleId) {
-        this.moduleId = moduleId;
-        resetAdmissionId();
-    }
-
-    /**
-     * Get timestamp
-     *
-     * @return timestamp
-     **/
     @ApiModelProperty()
     public LocalDateTime getTimestamp() {
         return this.timestamp;
     }
 
     public void setTimestamp(LocalDateTime timestamp) {
         this.timestamp = timestamp;
-        resetAdmissionId();
```

Comment:
can we really remove this here?
I don't know if that was necessary before, but if it was, it should stay.

Reply:
Yes, the timestamp or date is only necessary in the exam admission, this removal is done in the subcclass ExamAdmission

Reply:
This is the super-class of both exam admission and course admission. The update can be removed though because no admission id ever contains the admission timestamp! (The timestamp contained in the exam admission originates from the examId according to the API specification, if I am not mistaken)